.github/.OwlBot.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker:
+  image: "gcr.io/cloud-devrel-public-resources/owlbot-java:latest"
+
+deep-remove-regex:
+- "/grpc-google-.*/src"
+- "/proto-google-.*/src"
+- "/gapic-google-.*/src"
+
+deep-preserve-regex:
+- "/google-.*/src/test/java/com/google/cloud/.*/v.*/it/IT.*Test.java"
+- "/gapic-google-cloud-storage-v2/src/main/resources/com/google/storage/v2/gapic_metadata.json"
```

Comment:
We don't want to leave the file that is there, we want it to be updated based on the file that is in googleapis/googleapis.

Reply:
Fixed this by copying file directly to resource directory from googleapis-gen. See .github/.OwlBot.yaml